node/malus/README.md

```diff
@@ -11,36 +11,33 @@ The first argument determines the behavior strain. The currently supported are:
 ## Integration test cases
 
 To define integration tests create file
-in the toml format as used with [gurke][gurke]
+in the toml format as used with [zombie-net][zombie-net]
 under `./integrationtests` with either extension
-`.toml` or `.toml.tera` depending on if you use
-tera based templating.
+`.toml` or `.toml.tera`(**NOT available yet**) depending on if you use tera based templating.
 
-> For the time being non-templated variants should be preferred!
 ## Usage
 
-> Assumes you already gained permissiones, followed the [GKE access guide][gke],
-> and you installed [gurke][gurke].
+> Assumes you already gained permissiones, ping in element @javier:matrix.parity.io to get access.
+> and you installed [zombie-net][zombie-net].
 
 To launch a test case in the development cluster use (e.g. for the  ./node/malus/integrationtests/0001-dispute-valid-block.toml):
 
 ```sh
-# declare the containers pulled in by gurke test definitions
-export SYNTHIMAGE=paritypr/synth-wave:3639-0.9.9-7edc6602-ed5fb773
-export COLIMAGE=paritypr/colander:3639-7edc6602
-export MALUSIMAGE=paritypr/malus:3639-7edc6602
-export SCRIPTSIMAGE=paritytech/simnet:v9
+# declare the containers pulled in by zombie-net test definitions
+export MALUSIMAGE=docker.io/paritypr/malus:4131-ccd09bbf
+export SYNTHIMAGE=docker.io/paritypr/synth-wave:4131-0.9.12-ccd09bbf-29a1ac18
```

Comment:
Can we think of sane name that's not obscure

Reply:
Can we use `POLKAIMAGE` or `HONESTIMAGE` :) ?

Reply:
I think the initial idea was to not make this to be confused as something that's supposed to be run in production, so it should include some *integration test* component.

Reply:
Oks, sounds good. Maybe going with something literal like `INTEGRATIONTESTIMAGE` ?

Reply:
`ZOMBIENET_INTEGRATION_TEST_IMAGE`

Reply:
Sounds good, I'm also super happy to add some `_` :)